test/python/transpiler/test_basic_mapper.py

```diff
@@ -28,7 +28,7 @@ def test_trivial_case(self):
 
          Coupling map: [1]--[0]--[2]
         """
-        coupling = Coupling({0: [1, 2]})
+        coupling = Coupling([(0, 1), (0, 2)])
```

Comment:
It seems the format specified in Coupling is list of lists, not list of pairs. 
Maybe a reason for this is JSON encoding. I'm not fully sure.
But until we discuss and potentially change that in all Coupling constructors, can you make your code here consistent with the Coupling definition?